PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -259,4 +259,4 @@ integration, telemetry connectivity, and Azure Toolkit integration.
   that require it. Otherwise, it is not necessary to authenticate non-Azure Open-AI clients.
   Please refer to
   the [KeyCredential Class documentation](https://learn.microsoft.com/java/api/com.azure.core.credential.keycredential?view=azure-java-stable)
-  for more information.
\ No newline at end of file
+  for more information.
```

Comment:
these strings are the exact same, including number of spaces so I'm not sure how it's in the diff

Reply:
figured out it comes from the line break so its not problematic